File: terraform/src/main/kotlin/com/hyfly/tf/actuator/entity/validate/Snippet.java
package com.hyfly.tf.actuator.entity.validate;

import com.alibaba.fastjson2.annotation.JSONField;
import lombok.Data;

@Data
public class Snippet {

    @JSONField(name = "context")
    private String context;

    @JSONField(name = "code")
    private String code;

    @JSONField(name = "start_line")
    private Integer startLine;
}

File: terraform/src/main/kotlin/com/hyfly/tf/actuator/entity/validate/Snippet.kt
package com.hyfly.tf.actuator.entity.validate

import com.alibaba.fastjson2.annotation.JSONField

class Snippet {
    @JSONField(name = "context")
    val context: String? = null

    @JSONField(name = "code")
    val code: String? = null

    @JSONField(name = "start_line")
    val startLine: Int? = null
}
